depthai_ros_driver/include/depthai_ros_driver/device_ros.hpp

```diff
@@ -1,10 +1,12 @@
 #include <depthai_ros_driver/dai_utils.hpp>
+#include <depthai_ros_driver/conversion.hpp>
```

Comment:
Thoughts on moving this to depthai_*_msgs?
cc: @kota-rr

Reply:
I don't have strong opinions. Keeping `conversion` to the driver reduces the dependency of `msgs` package, but it makes sense the conversion is a part of the messages.

Reply:
We'll move it to msgs or a conversion package later on in that case.